tests/unit/Change.spec.ts

```diff
@@ -242,8 +263,66 @@ describe('Change component', () => {
     wrapper.destroy()
   })
 
-  it('renders Change view', () => {
+  it('renders Change view business information page', () => {
+    expect(wrapper.findComponent(Change).exists()).toBe(true)
+
+    // Business information page components
+    expect(wrapper.findComponent(YourCompany).exists()).toBe(true)
+    expect(wrapper.findComponent(PeopleAndRoles).exists()).toBe(true)
+
+    // Review and confirm page components
+    expect(wrapper.findComponent(ChangeSummary).exists()).toBe(false)
+    expect(wrapper.findComponent(DocumentsDelivery).exists()).toBe(false)
+    expect(wrapper.findComponent(CompletingParty).exists()).toBe(false)
+    expect(wrapper.findComponent(CertifySection).exists()).toBe(false)
+  })
+
+  it('renders Change view review and confirm page', async () => {
     expect(wrapper.findComponent(Change).exists()).toBe(true)
+    store.state.stateModel.summaryMode = true
+    wrapper.vm.showFee = true
```

Comment:
Changing the computed value triggers a re-rendering so a wait was necessary,

Reply:
Please add a code comment explaining why the wait.